yacl/crypto/primitives/psu/krtw19_psu.cc

```diff
@@ -0,0 +1,228 @@
+// Copyright 2024 zhangwfjh
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "yacl/crypto/primitives/psu/krtw19_psu.h"
+
+#include <algorithm>
+#include <array>
+#include <iterator>
+#include <random>
+#include <unordered_set>
+#include <utility>
+
+#include "yacl/utils/serialize.h"
+
+namespace yacl::crypto {
+
+namespace {
+
+// reference: https://eprint.iacr.org/2019/1234.pdf (Figure 2)
+constexpr float ZETA{0.06f};
```

Comment:
Style: would you like to rename all `constexpr` var to `kXXXX` (e.g. `ZETA` to `kZeta`), capitalizing only the first letter of each word.